src/cow-react/modules/limitOrders/hooks/useLimitOrdersFullState.ts

```diff
@@ -0,0 +1,24 @@
+import { useAtomValue, useUpdateAtom } from 'jotai/utils'
+import {
+  limitOrdersAtom,
+  LimitOrdersFullState,
+  limitOrdersFullStateAtom,
```

Comment:
Another part that's a bit confusing.

`limitOrdersAtom` is the raw state? Can you update the atom name to match the type etc?

Reply:
Yep, thank you!